parm/config/config.resources

```diff
@@ -40,87 +40,52 @@ if [ $step = "prep" -o $step = "prepbufr" ]; then
 
 elif [ $step = "anal" ]; then
 
-    export wtime_anal="02:30:00"
+    export wtime_anal="06:00:00"
+    export npe_anal=800
+    if [ $CASE = "C384" -o $CASE = "C192" -o $CASE = "C96" -o $CASE = "C48" ]; then export npe_anal=84; fi
+    export nth_anal=4  
+    if [[ "$machine" = "WCOSS_DELL_P3" ]]; then export nth_anal=7; fi
+    export npe_node_anal=$(echo "$npe_node_max / $nth_anal" | bc)
     export nth_cycle=$npe_node_max
-    export npe_anal=160
-    export npe_node_anal=2
-    export nth_anal=12
-    if [[ "$machine" = "WCOSS_DELL_P3" ]]; then
-      export nth_anal=14
-    fi
-    if [[ "$machine" != "WCOSS_C" ]]; then
-     if [ $CASE = "C384" -o $CASE = "C192" -o $CASE = "C96" -o $CASE = "C48" ]; then
-      export npe_anal=84
-      export nth_anal=4
-      export npe_node_anal=$(echo "$npe_node_max / $nth_anal" | bc)
-      if [[ "$machine" = "WCOSS_DELL_P3" ]]; then
-        export npe_node_anal=7
-      fi
-     fi
-    fi
-    if [[ "$machine" == "WCOSS_C" ]]; then
-        export memory_anal="3072M"
-    fi
+    if [[ "$machine" == "WCOSS_C" ]]; then export memory_anal="3072M"; fi
 
 elif [ $step = "gldas" ]; then
 
     export wtime_gldas="02:00:00"
     export npe_gldas=96  
     export nth_gldas=1 
-    export npe_node_gldas=24
+    export npe_node_gldas=$npe_node_max
     export npe_gaussian=96  
     export nth_gaussian=1 
-    export npe_node_gaussian=24  
-    if [[ "$machine" = "WCOSS_DELL_P3" ]]; then
-        export npe_gldas=112 
-        export npe_node_gldas=28
-    fi
-    if [[ "$machine" == "WCOSS_C" ]]; then
-        export memory_gldas="3072M"
-    fi
+    export npe_node_gaussian=24           
+    if [[ "$machine" = "WCOSS_DELL_P3" ]]; then export npe_gldas=112 ; fi
+    if [[ "$machine" == "WCOSS_C" ]]; then export memory_gldas="3072M"; fi
 
 elif [ $step = "fcst" ]; then
 
-    if [[ "$machine" == "WCOSS_DELL" ]]; then
-      export npe_node_max=24
-    fi
     export wtime_fcst="01:00:00"
-    export wtime_fcst_gfs="06:00:00"
+    export wtime_fcst_gfs="10:00:00"
```

Comment:
Hera's max walltime is 8hrs so this won't work on Hera. Perhaps we need to introduce a wtime_max variable for each machine and use it here?

Reply:
reduced to 8 hrs.  Should use much less, but this is to allow the fcst to finish if low resources are given.